packages/core/src/routes/application.ts

```diff
@@ -106,4 +108,21 @@ export default function applicationRoutes<T extends AuthedRouter>(router: T) {
       return next();
     }
   );
+
+  router.get(
+    '/applications/get-started/:library/:locale',
+    koaGuard({ params: object({ library: string(), locale: string() }) }),
+    async (ctx, next) => {
+      const {
+        params: { library, locale },
+      } = ctx.guard;
+
+      const markdownFiles = await findGetStartedMarkdownsByLibraryAndLocale(library, locale);
```

Comment:
do we need to handle the `findGetStartedMarkdownsByLibraryAndLocale`  internal exceptions?

Reply:
I'm not sure. If we need to handle the exceptions, like in a  file not found situation, where should I write the error handling code? I don't see any of the APIs in the routes has error handling